rw-system.sh

```diff
@@ -210,7 +210,7 @@ if getprop ro.vendor.build.fingerprint | grep -q -i \
     -e xiaomi/clover -e xiaomi/wayne -e xiaomi/sakura \
     -e xiaomi/nitrogen -e xiaomi/whyred -e xiaomi/platina \
     -e xiaomi/ysl -e nubia/nx60 -e nubia/nx61 -e xiaomi/tulip \
-    -e xiaomi/lavender -e xiaomi/olivelite -e xiaomi/pine; then
+    -e xiaomi/lavender - e xiaomi/olive -e xiaomi/olivelite -e xiaomi/pine; then
```

Comment:
Will that work with that space between '-' and 'e'?

Reply:
Fuuuccckkk
Thanks